core/internal/corelib/summarylib.go

```diff
@@ -3,7 +3,8 @@ package corelib
 import (
 	"fmt"
 
-	"github.com/segmentio/encoding/json"
+	// "github.com/segmentio/encoding/json"
```

Comment:
better to just remove rather than comment out, we can always add it back in

Reply:
oh yeah, part of the same thing as your first comment, just left it for easier switch over and never cleaned the code properly. Thanks!